openapi.yaml

```diff
@@ -1,279 +1,739 @@
+            The default distance unit is mile
+          type: number
+          format: float
+          example: 1.8
+        tags:
+          description: Tags associated with this location
+          type: array
+          items:
+            type: string
+            enum:
+              - building
+              - dining
+              - cultural-center
+              - parking
+              - other
+        parkingZoneGroup:
+          description: Zonegroup of location if location is parking
+          type: string
+          enum:
+            - A1
+            - A2
+            - A3
+            - B1
+            - B2
+            - B3
+            - C
+            - R
+        evParkingSpaceCount:
+          description: Count of electric vehicle parking spaces if the location is parking
+          type: number
+          format: integer
+        adaParkingSpaceCount:
+          description: Count of ADA parking spaces if the location is parking
+          type: number
+          format: integer
+        motorcycleParkingSpaceCount:
+          description: Count of motorcycle parking spaces if the location is parking.
+          type: number
+          format: integer
+        weeklyMenu:
+          description: Link to a weekly menu if the location is dining.
+          type: string
+          format: url
+          example: https://app.uhds.oregonstate.edu/api/dining/weeklymenu/index?loc=1
+    ServicesAttributes:
+      properties:
+        name:
+          type: string
+          description: Name of the Service.
+        tags:
+          description: Tags associated with this location
+          type: array
+          items:
+            type: string
+            example: recreation
+        openHours:
+          $ref: '#/components/schemas/DayHours'
+    RelationshipsInLocations:
+      properties:
+        services:
+          $ref: '#/components/schemas/ServicesRelationship'
+    RelationshipsInServices:
+      properties:
+        locations:
+          $ref: '#/components/schemas/ServicesRelationship'
+    ServicesRelationship:
+      properties:
+        data:
+          type: array
+          items:
+            $ref: '#/components/schemas/ResourceObjectIdentifier'
+        links:
+          $ref: '#/components/schemas/SelfLink'
+    ResourceObjectIdentifier:
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+    DayHours:
+      description: >-
+        This array represents open hours for a location, relative to the date
+        that the API returns the response. The first element represents the current day,
+        the second represents the next day etc. All times are in UTC.
+      type: array
+      items:
+        $ref: '#/components/schemas/TimeSlot'
+    TimeSlot:
+      properties:
+        start:
+          type: string
+          format: date-time
+        end:
+          type: string
+          format: date-time
```

Comment:
Here's the new `DayHours`. I didn't  add an additional timestamp for the days because the `start` and `end` values are already in timestamp format.